debuild/postinst

```diff
@@ -1,7 +0,0 @@
-if dpkg-maintscript-helper supports rm_conffile 2>/dev/null; then
```

Comment:
I actually have no idea where this code snippet came from... Maybe one of the deb tools I was playing with generated it?